rest/rest-resources/src/main/resources/OSGI-INF/blueprint/service.xml

```diff
@@ -29,58 +29,78 @@ limitations under the License.
 
     <cxf:bus>
         <cxf:features>
+<!--            <bean class="org.apache.cxf.interceptor.security.JAASAuthenticationFeature">
+                    <property name="reportFault" value="true"/>
+            </bean>-->
             <cxf:logging/>
         </cxf:features>
     </cxf:bus>
 
     <reference id="localManagementContext"
                interface="org.apache.brooklyn.api.mgmt.ManagementContext" />
+    <reference id="localManagementContextInternal"
+               interface="org.apache.brooklyn.core.mgmt.internal.ManagementContextInternal" />
 
     <bean id="accessResourceBean" class="org.apache.brooklyn.rest.resources.AccessResource">
         <property name="managementContext" ref="localManagementContext" />
+        <property name="managementContextInternal" ref="localManagementContextInternal" />
```

Comment:
Following the previous comment, why not inject `localManagementContextInternal` in the `managementContext` property?